### ISSUE TITLE
nomer needs a logo

### ISSUE THREAD

**Issue by seltmann** [globalbioticinteractions/nomer#85]
I am putting together a presentation for the Entomological Collections workshop and thought how nice it would be if nomer had a "face" or a logo to include on slides.

**Comment by jhpoelen** [comment]
Yes! I can see how giving Nomer a face would help to talk about them! Any ideas?
